lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,19 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "partially-correct"
+    notification_icon = "asterisk"
+    notification_message = "Partially correct (3/4 points earned)"
+%>
+
+<div class="notification ${notification_type}">
+    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
+    <span class="notification-message">${notification_message}</span>
+    % if hint_notification:
+    <button class="btn btn-default btn-small notification-btn">${_('Next Hint')}</button>
+    % endif
+    <button class="btn btn-default btn-small notification-btn review-btn sr">${_('Review')}</button>
+</div>
```

Comment:
Please do end files with a newline. If nothing else, it prevents this red glyph. :)